Makefile

```diff
@@ -2,7 +2,7 @@
 OPEN_API_URL = https://raw.githubusercontent.com/openfga/api/main/docs/openapiv2/apidocs.swagger.json
 OPENAPI_GENERATOR_CLI_DOCKER_TAG = v6.0.1
 NODE_DOCKER_TAG = 16-alpine
-GO_DOCKER_TAG = 1
```

Comment:
Why was this updated? We are always targeting the latest go version

Reply:
revert the change